js/apps/viewer/src/app/scene/scene.tsx

```diff
@@ -498,6 +498,7 @@ export class SceneModels extends PureComponent<Props, State> {
                         return {clipPlanes};
                     });
                 }),
+            // TODO: Remove this one!
```

Comment:
TODO still valid?

Reply:
Yes, I wasn't sure how to handle that right now.